lib/features/family/features/profiles/screens/profile_selection_screen.dart

```diff
@@ -92,28 +92,6 @@ class _ProfileSelectionScreenState extends State<ProfileSelectionScreen> {
           } else {
             await context.pushNamed(FamilyPages.childrenOverview.name);
           }
-        } else if (state is ProfilesNeedsRegistration) {
```

Comment:
This part is not needed because of the fix in approuter and removing the redirection screen

Reply:
Noice.